app/controllers/request-to-go-live/organisation-address/get.controller.js

```diff
@@ -11,6 +11,8 @@ const formatServicePathsFor = require('../../../utils/format-service-paths-for')
 module.exports = function getOrganisationAddress (req, res) {
   const isRequestToGoLive = Object.values(paths.service.requestToGoLive).includes(req.route && req.route.path)
 
+  const isStripeUpdateOrgDetails = paths.account.yourPsp.stripeSetup.updateOrgDetails.includes(req.route && req.route.path)
```

Comment:
Can the following new routes be restricted to stripe accounts only ? Middleware `restrictToStripeAccountContext` can be used for these.

```
checkOrgDetails: '/your-psp/:credentialId/check-organisation-details',
updateOrgDetails: '/your-psp/:credentialId/update-organisation-details'
```

Reply:
Done - thanks